backend/src/seed/factories/users.js

```diff
@@ -24,7 +24,7 @@ export default function create() {
       }
       args = await encryptPassword(args)
       const user = await neodeInstance.create('User', args)
-      const email = await neodeInstance.create('EmailAddress', { email: args.email })
+      const email = await factoryInstance.create('EmailAddress', { email: args.email })
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-09T07:05:14Z" title="Monday, September 9th 2019, 9:05:14 am +02:00">Sep 9, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -24,7 +24,7 @@ export default function create() {
       }
       args = await encryptPassword(args)
       const user = await neodeInstance.create('User', args)
-      const email = await neodeInstance.create('EmailAddress', { email: args.email })
+      const email = await factoryInstance.create('EmailAddress', { email: args.email })
```


what's the difference between `factoryInstance` and `neodeInstance`?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-09-10T13:23:12Z" title="Tuesday, September 10th 2019, 3:23:12 pm +02:00">Sep 10, 2019</time>_

---


We might want to refactor this with https://github.com/aexmachina/factory-girl (although the repo looks rather old).

`factoryInstance` is a reference to the other factories. If you would do `factoryInstance.create(..)` that would give you all the defaults and also setup associations. If you use `neodeInstance` you would end up with a `User` without `EmailAddress` etc.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-10T15:35:21Z" title="Tuesday, September 10th 2019, 5:35:21 pm +02:00">Sep 10, 2019</time>_

---


ahhh, ok! great... thanks